src/lev-report/index.js

```diff
@@ -67,12 +62,12 @@ class LevReport extends React.Component {
           updateTo={this.updateTo.bind(this)}
           groups={this.state.groups}
           currentGroup={this.state.currentGroup}
+          withoutGroups={this.state.withoutGroups}
         />
-        <CountDiv>
-          <UsageCounter count={this.state.total}
-                        from={this.state.from} to={this.state.to}
-                        group={this.state.currentGroup}/>
-        </CountDiv>
+        <UsageCounter count={this.state.total}
+                      from={this.state.from} to={this.state.to}
+                      group={this.state.currentGroup}
+                      withoutGroups={this.state.withoutGroups}/>
```

Comment:
Any reason you're providing `withoutGroups` to the `UsageCounter`, it doesn't seem to be necessary?

Reply:
Removed